dynawo/sources/Modeler/ModelManager/DYNModelManager.cpp

```diff
@@ -1151,6 +1167,7 @@ ModelManager::printInitValues(const string & directory) {
     file << std::setw(50) << std::left << xAliasesNames[i].first << std::right << ": alias of " << xAliasesNames[i].second << "\n";
```

Comment:
Do we go through this method if we don't ask for initDumpFiles? If it is not the case, where is done the calculated variables evaluation after the local initialization? (for example for iRotor0Pu that needs to be evaluated)

Reply:
Line 920, right before the injection in the dynamic model